src/extension.ts

```diff
@@ -81,6 +86,32 @@ class SonicPi {
             flashDecorationType.dispose();
         }, 250);
     }
+
+    private async getDynamicPort() {
+        let ports = { server_listen_port: 4557, gui_listen_port: 4558 };
+        const userHome = require('user-home');
```

Comment:
I want to use `import`statement and move this line to top of this file.
(sorry, I missed here at first review)

Reply:
thanks, fixed.